apps/web/src/components/team-builder/v2/builder.module.css

```
@@ -861,6 +894,29 @@
   width: 100%;
   overflow: hidden;
   cursor: pointer;
+}
+
+/* Tighten BP / Acc widths at narrow slot widths so move names get more
+   room before truncating. The "BP" / "ACC" labels are still rendered
+   but the value column fits a 3-digit BP / 4-char "100%" without
+   reserving extra space. */
+@container slot (max-width: 1239px) {
+  :global(.mvline) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)